datafusion/core/tests/sqllogictests/test_files/join_disable_repartition_joins.slt

```diff
@@ -56,11 +56,13 @@ Limit: skip=0, fetch=5
 ----------TableScan: annotated_data projection=[a, c]
 physical_plan
 GlobalLimitExec: skip=0, fetch=5
---ProjectionExec: expr=[a@1 as a]
-----CoalesceBatchesExec: target_batch_size=8192
-------HashJoinExec: mode=CollectLeft, join_type=Inner, on=[(c@0, c@1)]
---------CsvExec: file_groups={1 group: [[WORKSPACE_ROOT/datafusion/core/tests/data/window_2.csv]]}, projection=[c], has_header=true
---------CsvExec: file_groups={1 group: [[WORKSPACE_ROOT/datafusion/core/tests/data/window_2.csv]]}, projection=[a, c], output_ordering=[a@0 ASC NULLS LAST], has_header=true
+--SortPreservingMergeExec: [a@0 ASC NULLS LAST], fetch=5
```

Comment:
I agree this plan looks better, but on the other hand it is part of a test that is supposed to disable repartitioning join inputs 🤔 

```
set datafusion.optimizer.repartition_joins = false;
```

But I think it is ok, because `datafusion.optimizer.repartition_joins` controls repartitioning the left (build) side which is not repartitioned